API.md

```diff
@@ -39,6 +39,27 @@ This page documents the Sample service API endpoints. Apart from the Service Inf
 
 An `HTTP 201 Created` status code is returned if the sample unit request creation was a success. An `HTTP 400 Bad Request` is returned if any of the required parameters are missing, or if a sample unit request already exists for the same criteria.
 
+## Get Sample Unit Size Request
+* `POST /samples/sampleunitsize` gets the total number of sample units.
```

Comment:
This will need changing if we agree it should be a GET method call

Reply:
GET with body doesn't make any sense and it's not supported very well by Spring MVC